content/docs/reference-dom-elements.md

```diff
@@ -48,23 +48,23 @@ function MyComponent() {
 
 ### htmlFor {#htmlfor}
 
-Since `for` is a reserved word in JavaScript, React elements use `htmlFor` instead.
+Jako że `for` jest zarezerwowanym słowem w JavaScripcie, React używa zamiast niego `htmlFor`.
 
 ### onChange {#onchange}
 
-The `onChange` event behaves as you would expect it to: whenever a form field is changed, this event is fired. We intentionally do not use the existing browser behavior because `onChange` is a misnomer for its behavior and React relies on this event to handle user input in real time.
+Zdarzenie `onChange` zachowuj się zgodnie z oczekiwaniami: jest aktywowane za każdym razem, gdy zmienia się wartość w polu formularza. Specjalnie nie używamy implementacji przeglądarkowej, ponieważ `onChange` ma nazwę nieadekwatną do zachowania. React natomiast używa tego zdarzenia do obsłużenia w czasie rzeczywistym zmian w danych wejściowych użytkownika.
 
 ### selected {#selected}
 
-The `selected` attribute is supported by `<option>` components. You can use it to set whether the component is selected. This is useful for building controlled components.
+Atrybut `selected` jest wspierany przez element `<option>`. Możesz go użyć do sprawdzenia, czy dany element listy został wybrany. Przydaje się do budowania kontrolowanych komponentów.
 
 ### style {#style}
 
->Note
+>Wskazówka
 >
->Some examples in the documentation use `style` for convenience, but **using the `style` attribute as the primary means of styling elements is generally not recommended.** In most cases, [`className`](#classname) should be used to reference classes defined in an external CSS stylesheet. `style` is most often used in React applications to add dynamically-computed styles at render time. See also [FAQ: Styling and CSS](/docs/faq-styling.html).
+>Niektóre przykłady w dokumentacji używają `style` dla wygody, ale **używanie atrybutu `style` do stylowania elementów ogólnie nie jest zalecane.** W większości przypadków należy korzystać z atrybutu [`className`](#classname), odnosząc się do klas definiowanych w zewnętrznym arkuszu stylów (ang. *stylesheet*). `style` jest najczęściej używany w aplikacjach reactowych do dodawania stylów dynamicznych, zmieniających się przy kolejnych renderowaniach. Zobacz też [FAQ: Stylowanie i CSS](/docs/faq-styling.html).
```

Comment:
Może nie pomijajmy "primary means of styling"? Poza tym "stylowanie elementów"? Może jednak "formatowanie wyglądu".

```suggestion
>Niektóre przykłady w dokumentacji używają `style` dla wygody, ale **częste stosowanie atrybutu `style` do formatowanie wyglądu elementów ogólnie nie jest zalecane.** W większości przypadków należy korzystać z atrybutu [`className`](#classname), odnosząc się do klas definiowanych w zewnętrznym arkuszu stylów (ang. *stylesheet*). `style` jest najczęściej używany w aplikacjach reactowych do dodawania stylów dynamicznych, zmieniających się przy kolejnych renderowaniach. Zobacz też [FAQ: Stylowanie i CSS](/docs/faq-styling.html).
```

Reply:
Wiem, że to razi, ale tak się już utarło w języku programistycznym - "stylowanie komponentów". Ostatecznie dałbym po prostu "określania wyglądu".
Tak czy siak, masz literówkę - powinno być "formatowani**a**".